dev-libs/libsodium/libsodium-1.0.16.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Firstly, please don't change the old versions and instead just add it to the newest.

Secondly, you've got wrong version in the commit message. Also, we don't add version to the prefix; instead, mention it in the commit message. However, if you only add it to the newest then there's no need for version at all.